src/index.html

```diff
@@ -6,9 +6,15 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Stop watch</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="style.css">
```

Comment:
in these tasks you no longer need to manually compile styles from `SCSS` -> `CSS`. To do this, we have npm packages - the `npm i` command and the project start command - `npm start`.